gammapy/modeling/models/spectral.py

```diff
@@ -1533,6 +1536,105 @@ def from_region_map(cls, map, **kwargs):
         return cls(energy=energy, values=values, **kwargs)
 
 
+class TemplateNDSpectralModel(SpectralModel):
+    """A model generated from a ND array where extra dimensions define the parameter space.
+
+    For more information see :ref:`templateND-spectral-model`.
+
+    Parameters
+    ----------
+    map : `~gammapy.maps.RegionNDMap`
+        Map template. 
+    meta : dict, optional
+        Meta information, meta['filename'] will be used for serialization
+    interp_kwargs : dict
+        Interpolation keyword arguments passed to `gammapy.maps.Map.interp_by_coord`.
+        Default arguments are {'interp': 'linear', 'fill_value': 0}.
+    """
+
+    tag = ["TemplateNDSpectralModel", "templateND"]
+
+    def __init__(self, map, interp_kwargs=None, meta=None, filename=None):
+        self._map = map.copy()
+        self.meta = dict() if meta is None else meta
+        if filename is not None:
+            filename = str(make_path(filename))
+        self.filename = filename
+
+        points_scale = [
+            "lin",
+            "lin",
+            "log",
+        ]
+        parameters = []
+        for axe in map.geom.axes:
+            if axe.name not in ["energy_true", "energy"]:
+                center = (axe.bounds[1] + axe.bounds[0]) / 2
+                parameter = Parameter(
+                    name=axe.name,
+                    value=center,
+                    unit=axe.unit,
+                    scale_method="scale10",
+                    min=axe.bounds[0],
+                    max=axe.bounds[-1],
+                    interp="lin",
+                )
+                points_scale.append("lin")
+                parameters.append(parameter)
+        self.default_parameters = Parameters(parameters)
```

Comment:
I suppose an `amplitude` parameter is required. It probably needs to be added to the list of tabulated parameters.

Reply:
> Thanks @QRemy ! This looks good, but I have a general comment.
> 
> Our `SpectralModel` are usually either regular spectra with an `amplitude` parameter (having flux dimension, an additive model) or a `NormSpectralModel` with a `norm` parameter (dimensionless, a multiplicative model). Here the `Model` can be both, or it can have neither `amplitude` nor `norm` parameter. This has to be clarified. Possibly by testing the existence of an `amplitude` or `norm` parameter or by creating two distinct types of models.

The behaviour is the same for the `TemplateSpectralModel`, it has no `amplitude` or `norm` parameter and technically it could be two distinct types of models (normed or not).

Reply:
The behaviour is the same for the `TemplateSpectralModel`, it has no amplitude or norm parameter